### ISSUE TITLE
Semi-undefined experimental WebRTC obscura

### ISSUE THREAD

**Issue by eyedeekay** [eyedeekay/reseed-tools#7]
Ties in to [Reseed-over-WebRTC Datachannel](https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/issues/80) and also the reseed-over-WebRTC datachannel client proxy which I don't think I have even started yet. The concept would basically be to create a network of browsers, collaborating to provide reseeds by proxy(over WebRTC) to participants who cannot reach a regular reseed server or may wish to avoid their ISP having visibility into when they did a reseed.

**Comment by backkem** [comment]
If you haven't already, check out [pion/webrtc](https://github.com/pion/webrtc). It's a WebRTC stack in pure Go. You can run it naively and even in the browser using WASM. If you have any questions, the #pion channel of the gopher slack hosts a vibrant community that loves to discuss all things WebRTC.

**Comment by eyedeekay** [comment]
Absolutely, I'm a big fan of the work, will definitely be using pion/webrtc when I finally implement this. Still quite a distance from tackling this issue at the moment though, I basically understand WebRTC and what it's for, but I lack experience actually implementing things with it. I'll definitely be in touch about it when I am in the implementation phase though.
